src/openforms/contrib/eherkenning/utils.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from django.templatetags.static import static
+
+from openforms.authentication.base import LoginLogo
+
+
+def get_eherkenning_logo(request, label) -> Optional[LoginLogo]:
```

Comment:
see similar remark with the digid util

I'd also just put `digid_eherkenning` together in `openforms.contrib`, you basically always consider those two together for Dutch gov auth.